examples/Boost-thread/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# download boost
+hunter_add_package(Boost COMPONENTS system thread)
```

Comment:
Actually I vote for not mixing unrelated stuff. E.g. we can just print thread-id to std::cout, no need to use chrono.